pkg/driverbuilder/builder/templates/flatcar.sh

```diff
@@ -20,9 +20,10 @@ rm -Rf /tmp/kernel
 mkdir -p /tmp/kernel
 mv /tmp/kernel-download/*/* /tmp/kernel
 
-curl --silent -o /tmp/kernel.config -SL {{ .KernelConfigURL }}
```

Comment:
No need to download the kernel config file. We already have it passed by user/config ;)